src/content/meta/glossary.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Glossary
+description: A page with the term definitions Giant Swarm uses in the documentation.
+search: false
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - "What is a cloud-native developer platform?"
+  - "What is a developer environment?"
+  - "What is a management cluster?"
+  - "What is a managed app?"
+  - "What is the platform API?"
+  - "What is a platform operator?"
+  - "What is smart platform engineering?"
+  - "What is a workload cluster?"
+---
+
+We use a set of terms across our documentation that could be unclear to readers. In the following article we define those terms to clarify their meaning in the context of Giant Swarm.
+
+## Cloud native developer platform
+
+A cloud-native developer platform is an integrated collection of capabilities exposed through intuitive interfaces that facilitate the application development lifecycle. It offers flexible configuration to build a system that encodes your company practices. It integrates with different open source solutions to enable automation, connectivity, observability, and security for your workloads.
+
+## Developer environment
+
+In our context, a developer environment is the entity that defines the space where the developers will deploy their applications. It can be mapped to a Kubernetes namespace(s) or a cluster. The Platform capabilities like access management, security or observability are related to those environments.
+
+## Management cluster
+
+It is a Kubernetes cluster that acts a central point of the platform. It is in charge of hosting platform wide services like observability or security to ensure visualization and compliance over all the workloads in all clusters. [Platform operator](#platform-operator) have access to the its [API](#platform-api) to manage [workload clusters](#workload-cluster) and [managed apps](#managed-apps).
+
+## Managed app
+
+The managed apps are a set of curated diverse components that can be used in the [platform](#cloud-native-developer-platform) to enable capabilities across the [developer environments](#developer-environment). As an example, in case the developers want to allocate dynamically DNS records for their services they can leverage in [external-dns](https://github.com/giantswarm/external-dns-app/). The managed apps are operated by us and we provide regular upgrades to avoid security vulnerabilities and bugs.
```

Comment:
> curated diverse

🤔 

```suggestion
The managed apps are a set of curated components that can optionally be used in the [platform](#cloud-native-developer-platform) to enable diverse capabilities across the [developer environments](#developer-environment). As an example, in case the developers want to dynamically allocate DNS records for their services, they can leverage [external-dns](https://github.com/giantswarm/external-dns-app/). The managed apps are operated by us and we provide regular upgrades to avoid security vulnerabilities and bugs.
```

Also "operated by us" is something we should clarify now or later.